.github/workflows/precommit.yml

```diff
@@ -231,15 +231,15 @@ jobs:
       - name: Install tox
         run: python -m pip install tox
       - if: ${{ matrix.python-version == '3.8' && contains('macos', matrix.os.download_name) }}
-        name: Run tests (including doctest Tests)
-        run: python -m tox -e "py${PYTHON_VERSION/\./}-udf"
+        name: Run doctests
+        run: python -m tox -e "py${PYTHON_VERSION}-doctest-notudf"
```

Comment:
are you separating tests to be only doc tests vs only integ+unit

Reply:
Yes, with our current configuration, `coverage` will fail to collect measurements when doctests are run in parallel, I spent some time trying to fix this but couldn't find a solution for now. As a result, doctests are run sequentially and the other tests (integ, unit) can be parallelized.